pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -507,6 +507,19 @@ func NewPrometheusRuleSpec(ns string) *v1.PrometheusRuleSpec {
 				operatorHealthImpactLabelKey: "none",
 			},
 		},
+		{
+			Alert: "VirtualMachineCRCErrors",
+			Expr:  intstr.FromString("(kubevirt_vm_persistentvolume_info{volumeAttributes=~'.*mounter=rbd.*'} or ( kubevirt_vm_persistentvolume_info{volumeAttributes=~'.*rbd.csi.ceph.com.*'} and kubevirt_vm_persistentvolume_info{volumeAttributes!~'.*mounter=.*'} )) and kubevirt_vm_persistentvolume_info{volumeMode='Block', volumeAttributes!~'.*krbd:rxbounce.*'}"),
+			Annotations: map[string]string{
+				"description": "VirtualMachine {{ $labels.name }} may report OSD errors",
+				"summary":     "When running VMs using ODF storage with 'rbd' mounter or 'rbd.csi.ceph.com provisioner', it will report bad crc/signature errors and cluster performance will be severely degraded if krbd:rxbounce is not set.",
+				"runbook_url": fmt.Sprintf(runbookURLTemplate, "VirtualMachineCRCErrors"),
+			},
+			Labels: map[string]string{
+				severityAlertLabelKey:        "warning",
+				operatorHealthImpactLabelKey: "none",
+			},
+		},
```

Comment:
Sorry for missing the community meeting where this was discussed.
Catching up now I see the main claim is that provisioner-specific parts do not belong in kubevirt,
which I think makes sense.

But I do think that the PR content here is useful, as in, these PV metrics are usable
outside of alerting about this specific CRC error.

What about only taking out the provisioner-specific stuff to a more
opinionated component? and which component would that be?
I know we have RHEL6 specific alerts in SSP:
https://github.com/kubevirt/ssp-operator/pull/486

Reply:
I just opened a new PR in cdi to add the metric we need which are the most  controversial addition: https://github.com/kubevirt/containerized-data-importer/pull/2835

Since all PVCs that we need to fetch are labeled "app.kubernetes.io/managed-by: cdi-controller" I thought it made sense

Reply:
> metric we need which are the most controversial addition

Is the metric really controversial? if a PVC is part of a kubevirt VM, isn't it a viable kubevirt metric?
To me the controversial part is the highlighted chunk for this comment (+510 to +522)